libs/net/models/Areas/Subscriber/Models/Content/ContributorModel.cs

```diff
@@ -7,11 +7,7 @@ namespace TNO.API.Areas.Subscriber.Models.Content;
 /// </summary>
 public class ContributorModel : BaseTypeModel<int>
 {
-   #region Properties
-    /// <summary>
-    /// get/set - The primary key of the contributor.
-    /// </summary>
-    public int Id { get; set; }
```

Comment:
@jdtoombs Not sure how this snuck in with your PR.  I remember there was linting errors and I also posted a message on it.  Don't overwrite inherited properties or it causes issues.

I've removed them.